src/common/k8s-api/kube-watch-api.ts

```diff
@@ -147,11 +147,11 @@ export class KubeWatchApi {
       ? reaction(
         // Note: must slice because reaction won't fire if it isn't there
         () => [KubeWatchApi.context.contextNamespaces.slice(), KubeWatchApi.context.hasSelectedAll] as const,
-        ([namespaces, curSelectedAll], [, prevSelectedAll]) => {
+        ([namespaces, curSelectedAll], [prevNamespaces, prevSelectedAll]) => {
           if (curSelectedAll && prevSelectedAll) {
-            console.log(`[KUBE-WATCH-API]: Not changing watch for ${store.api.apiBase} because a new namespace was created but all namespaces are selected`);
+            const action = namespaces.length > prevNamespaces.length ? "created" : "deleted";
```

Comment:
getting fancy. Is it possible that one namespace is created and another deleted at the same time (wrt this reaction)?

Reply:
Well the message will still be correct on that case 😅.

I don't think begin so fancy is worth it.